app/services/import_planned_disbursements.rb

```diff
@@ -0,0 +1,37 @@
+require "csv"
+
+class ImportPlannedDisbursements
+  RODA_ID_KEY = "RODA identifier"
+  FORECAST_COLUMN_HEADER = /FC +(\d{4})\/\d{2} +FY +Q([1-4])/
+
+  def initialize(report:)
+    @report = report
+  end
+
+  def import(forecasts)
+    forecasts.each { |row| import_row(row) }
+  end
+
+  def import_row(row)
+    roda_identifier = row[RODA_ID_KEY]
```

Comment:
I'm not a huge fan of mutating objects that were passed in by the caller, unless the caller can guarantee it will never use those objects afterward. In this case it doesn't help us guarantee all the columns are processed, per my earlier comment.

Reply:
Fair enough - I didn't realise there'd be columns we wouldn't be using either, so getting rid of the ID isn't worth the effort